### PR TITLE
preselect highlight cell & fixed those warning

### PR DESCRIPTION
**Pull request by Ferrick90**

### CODE REVIEW

File: Example/FusumaExample/Sources/FSAlbumView.swift [hunk 1]
```diff
@@ -89,10 +89,10 @@ final class FSAlbumView: UIView, UICollectionViewDataSource, UICollectionViewDel
         if images.count > 0 {
             
             changeImage(images[0] as! PHAsset)
+            collectionView.reloadData()
+            collectionView.selectItemAtIndexPath(NSIndexPath(forRow: 0, inSection: 0), animated: false, scrollPosition: UICollectionViewScrollPosition.None)
```
Comment (ytakzk) [inline Example/FusumaExample/Sources/FSAlbumView.swift:93]: What's the purpose for this code? Is it needed?

```
collectionView.selectItemAtIndexPath(NSIndexPath(forRow: 0, inSection: 0), animated: false, scrollPosition: UICollectionViewScrollPosition.None)
```

**Comment by ytakzk** [comment]
@Ferrick90 Thank you for the pull request!
However, I left the warnings on purpose so that Fusuma can be built with Swift2.1 too. Is it possible to remove the codes for the warnings for now?

File: Example/FusumaExample/Sources/FSAlbumView.swift (hunk 1 above)
Comment (Ferrick90) [inline Example/FusumaExample/Sources/FSAlbumView.swift:93]: image
Once open gallery will select the first cell.

File: Example/FusumaExample/Sources/FSCameraView.swift [hunk 2]
```diff
@@ -84,7 +84,7 @@ final class FSCameraView: UIView, UIGestureRecognizerDelegate {
             
             // Focus View
             self.focusView         = UIView(frame: CGRect(x: 0, y: 0, width: 90, height: 90))
-            let tapRecognizer      = UITapGestureRecognizer(target: self, action:Selector("focus:"))
+            let tapRecognizer      = UITapGestureRecognizer(target: self, action: Selector("focus:"))
             tapRecognizer.delegate = self
```
Comment (ytakzk) [inline Example/FusumaExample/Sources/FSCameraView.swift:88]: @Ferrick90 You forgot this. When you turn this back to the original one, I will merge this pull request.

File: Sources/FSCameraView.swift [hunk 3]
```diff
@@ -84,7 +84,7 @@ final class FSCameraView: UIView, UIGestureRecognizerDelegate {
             
             // Focus View
             self.focusView         = UIView(frame: CGRect(x: 0, y: 0, width: 90, height: 90))
-            let tapRecognizer      = UITapGestureRecognizer(target: self, action: Selector("focus:"))
+            let tapRecognizer      = UITapGestureRecognizer(target: self, action:Selector("focus:"))
```
Comment (ytakzk) [inline Sources/FSCameraView.swift:87]: @Ferrick90 I am sorry but I mean you cannot use Selector for Swift 2.1

Comment (Ferrick90) [inline Sources/FSCameraView.swift:87]: @ytakzk In Swift 2.1 could write either "focus:" or Selector("focus:"). If I'm not wrong.

**Comment by ytakzk** [comment]
@Ferrick90 Yes, that's true. Thanks!

**Pull request merged**